MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java

```diff
@@ -0,0 +1,94 @@
+package cecelia.moodcookie;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Base64;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+
+import java.io.ByteArrayOutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+import cecelia.moodcookie.types.MySingleton;
+
+
+public class IndicoCrap extends Fragment {
+
+    public IndicoCrap() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        final View view = inflater.inflate(R.layout.fragment_indico_experimenting, container, false);
+
+        final String APIKey = BuildConfig.API_KEY;
+        final TextView output = (TextView) view.findViewById(R.id.sampleOutput);
+        String url = "https://apiv2.indico.io/fer"; //url for Indico's facial emotion recognition
+
+        StringRequest postRequest=new StringRequest(Request.Method.POST,url,new Response.Listener<String>() {
+            @Override
+            public void onResponse(String response) {
+                HashMap<String, Float> emotionDict = new HashMap(); //dictionary we store emotions and their values in
+                Float max_val = Float.parseFloat("0");
+                String max_emotion = "";
+                String shortenedResponse = response.substring(13, response.length()-2).replaceAll("\"",""); //strip "{"results": {", "}", and " from original response
```

Comment:
Not a fan of manually parsing JSON. Jackson and Gson are both good libraries for extracting this data.